src/battle/area/iwa/actor/buzzar.c

```diff
@@ -8,19 +8,19 @@
 
 #define NAMESPACE b_area_iwa_buzzar
 
-extern EvtScript N(init_8021B7E4);
-extern EvtScript N(takeTurn_8021BCD8);
-extern EvtScript N(idle_8021B930);
-extern EvtScript N(handleEvent_8021B940);
-extern EvtScript N(nextTurn_8021B918);
-extern EvtScript N(8021BEBC);
-extern EvtScript N(8021CCE8);
-extern EvtScript N(8021DF2C);
-extern EvtScript N(8021F030);
-extern EvtScript N(8021F7EC);
-extern EvtScript N(8021F808);
-extern EvtScript N(8021F824);
-extern EvtScript N(8021FA2C);
+extern EvtScript N(init);
+extern EvtScript N(taketurn);
+extern EvtScript N(idle);
+extern EvtScript N(handleEvent);
+extern EvtScript N(nextTurn);
+extern EvtScript N(attackWindBlast);
```

Comment:
I think we're going to use EVS_Attack_AttackName for these, so you're on the right track here.